crate/server/src/core/cover_crypt/update_policy.rs

```diff
@@ -265,3 +220,70 @@ async fn refresh_all_user_decryption_keys(
 
     Ok(())
 }
+
+/// Update a Covercrypt policy based on the specified action and master private key.
```

Comment:
It would be much clearer if you'd taken as argument a `Policy` instead of the master key. As of now, it is not clear form the documentation function whether or not the msk gets updated.

If you make this change, you may want to take ownership of the policy.